Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:xenial
 MAINTAINER Mitchell Hewes <me@mitcdh.com>
 
-ENV DRIVE_PATH="/drive"
+ENV DRIVE_PATH="/mnt/gdrive"
```

Comment:
for backwards compatibility, setting this to what it was hard coded as in the entrypoint script